From 9d1399e89d431b34a5a6e5e48ee40721c22815b8 Mon Sep 17 00:00:00 2001
From: Aurelien Jarno <aurelien@aurel32.net>
Date: Tue, 8 Mar 2016 15:16:40 +0100
Subject: [PATCH 10/20] Add sys/auxv.h wrapper to include/sys/

The GNU libc testsuite fails to build on powerpc/ppc64/ppc64le with the
following error:

    ../sysdeps/powerpc/test-get_hwcap.c:26:22: fatal error: sys/auxv.h: No such file or director

This is because test-get_hwcap.c includes <sys/auxv.h>, but we don't
provide a wrapper in include/sys. This patch adds one.

Changelog:
	* include/sys/auxv.h: New file.

(cherry picked from commit 0b8dedd38f304d796b6b9b349428bea7f1f7065f)
---
 ChangeLog          | 4 ++++
 include/sys/auxv.h | 1 +
 2 files changed, 5 insertions(+)
 create mode 100644 include/sys/auxv.h

diff --git a/ChangeLog b/ChangeLog
index 4a27eb7..3177332 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,7 @@
+2016-03-08  Aurelien Jarno  <aurelien@aurel32.net>
+
+	* include/sys/auxv.h: New file.
+
 2016-03-07  Hongjiu Zhang  <noctuorare@gmail.com>
 
 	* elf/sln.c (makesymlink): Change struct stat to stat64, and lstat
diff --git a/include/sys/auxv.h b/include/sys/auxv.h
new file mode 100644
index 0000000..dede2c3
--- /dev/null
+++ b/include/sys/auxv.h
@@ -0,0 +1 @@
+#include <misc/sys/auxv.h>
-- 
2.8.0

